portal-api/src/v1/api.js

```diff
@@ -316,19 +316,23 @@ const getUtilisationReports = async (bankId, options) => {
   }
 };
 
-const getUtilisationReportById = async (_id) => {
+/**
+ * @param {number} id
+ * @returns {Promise<import('@ukef/dtfs2-common/').UtilisationReportResponseBody>}
+ */
+const getUtilisationReportById = async (id) => {
   try {
-    if (!isValidMongoId(_id)) {
-      throw new Error(`Invalid MongoDB _id provided: '${_id}'`);
+    if (!Number.isInteger(id)) {
```

Comment:
I imagine this may be used elsewhere, so could we create a helper like `isValidMongoId` but for the SQL ids? It will probably just be a one liner which returns whether or not the value is an integer like this, but could increase the readability for why we're doing this check